src/context/WalletProvider/KeepKey/components/Label.tsx

```diff
@@ -4,8 +4,16 @@ import { RecoverDevice, ResetDevice } from '@shapeshiftoss/hdwallet-core'
 import { useRef, useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { Text } from 'components/Text'
+import { VALID_ENTROPY } from 'context/WalletProvider/KeepKey/components/RecoverySettings'
 import { useWallet } from 'hooks/useWallet/useWallet'
 
+const parseIntToEntropy = (entropy: string): RecoverDevice['entropy'] => {
+  const isValidEntropyNumber = (entropy: number): entropy is RecoverDevice['entropy'] =>
+    new Set(VALID_ENTROPY.map(parseInt)).has(entropy)
+  const parsedInt = parseInt(entropy)
```

Comment:
`parseInt` has some gotchas. I'd rather see

```suggestion
  const parsedInt = Math.floor(Number(entropy))
```

Reply:
Addressed in https://github.com/shapeshift/web/pull/1561/commits/d3474e4092cca0e2baad79bcc6fb05f1aba5d710.